Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs

```diff
@@ -537,13 +537,22 @@ protected virtual void OnSetDataState(Model.State state)
         // Updates UI based on values on View.Data
         protected virtual void UpdateUI()
         {
+            Debug.Log("TextElipsis: " + TextElipsis(Data.Title));
+            Debug.Log("Text base: " + Data.Title);
             titleText.text = TextElipsis(Data.Title);
-            if(brandText!=null) brandText.text = Data.Brand;
+            Debug.Log("Text in UI: " + titleText.text);            
+            if (brandText!=null) brandText.text = Data.Brand;
```

Comment:
We should decodeHTML on the brand too I think

Reply:
Ok!